src/Phoenix/PhoenixClient.ts

```diff
@@ -33,32 +35,24 @@ export class PhoenixClient {
     minTimeout: Notebook.retryAttemptDelayMs,
   };
 
-  public async allocateContainer(provisionData: IProvisionData): Promise<IResponse<IPhoenixConnectionInfoResult>> {
+  public async allocateContainer(provisionData: IProvisionData) {
```

Comment:
why remove the return type from this function and other functions?

Reply:
Now with the addition of diffrent error types in the return type we cant have a generic promise as return type.